gaffer-as-a-service/gaas-rest/pom.xml

```diff
@@ -36,6 +36,11 @@
         </dependencies>
     </dependencyManagement>
     <dependencies>
+        <dependency>
+            <groupId>uk.gov.gchq.gaffer</groupId>
+            <artifactId>gaas-common</artifactId>
+            <version>1.13.0</version>
+        </dependency>
```

Comment:
Versions for module specific dependencies should be stored in properties

Reply:
done